kv/db_test.go

```diff
@@ -20,19 +20,25 @@ package kv_test
 import (
 	"bytes"
 	"encoding/json"
+	"log"
 	"net/http"
 	"testing"
 
 	"github.com/cockroachdb/cockroach/client"
 	"github.com/cockroachdb/cockroach/kv"
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/testutils"
 	"github.com/cockroachdb/cockroach/util"
 	gogoproto "github.com/gogo/protobuf/proto"
 	yaml "gopkg.in/yaml.v1"
 )
 
 func createTestClient(addr string) *client.KV {
-	return client.NewKV(nil, client.CreateTestHTTPSender(addr))
+	httpClient, err := testutils.NewTestHTTPClient()
```

Comment:
Actually, I meant to go back to either have it return an error or take testing.T. I'm going for the latter.